alcs-frontend/src/app/shared/inline-datepicker/inline-datepicker.component.html

```diff
@@ -0,0 +1,27 @@
+<div>
+  <span *ngIf="!isEditing">
+    <span style="color: #666; font-style: italic" *ngIf="!selectedValue"> Not set </span>
+    <span *ngIf="selectedValue">
+      {{ formattedDate }}
+    </span>
+    <button style="height: 24px; width: 24px" mat-icon-button (click)="toggleEdit()">
```

Comment:
is it possible to have styles in scss instead?

Reply:
Done! Cleaned up here and also the intake.component.html